core/node/eth_sender/src/eth_fees_oracle.rs

```diff
@@ -40,6 +40,12 @@ impl GasAdjusterFeesOracle {
         previous_sent_tx: &Option<TxHistory>,
     ) -> Result<EthFees, EthSenderError> {
         let base_fee_per_gas = self.gas_adjuster.get_blob_tx_base_fee();
```

Comment:
it also makes sense to check if `base_fee_per_gas == 0` in `calculate_fees_no_blob_sidecar `